german/finger-ins-loch/finger-ins-loch.md

```diff
@@ -1,2 +1,4 @@
 Glückwunsch! Jetzt steckt ihr Finger im Loch fest! Leider wird der Wassereinstrom dadurch auch nur minimal aufgehalten.
 Ihnen sollte nun besser bald was einfallen, wie sie sich wieder befreien können...
+
+Ihnen sticht eine Kiste ins Auge. Sie öffnen sie, finden darin einen Lappen, ziehen den Finger aus dem Loch und stopfen den Lappen ins Loch. Nun müssen Sie schnellstmöglichst ans Ufer.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.